rpc-client-api/src/filter.rs

```diff
@@ -263,7 +264,8 @@ pub fn maybe_map_filters(
     node_version: Option<semver::Version>,
     filters: &mut [RpcFilterType],
 ) -> Result<(), String> {
-    if node_version.is_none() || node_version.unwrap() < semver::Version::new(1, 11, 2) {
+    let version_reqs = VersionReq::from_strs(&["<1.11.2", "~1.13"])?;
+    if node_version.is_none() || version_reqs.matches(&node_version.unwrap()) {
```

Comment:
nit: can we switch this to a `.map().unwrap_or()`.  the double-option-eval makes my eye twitch